lib/classes/Service.test.js

```diff
@@ -709,6 +712,14 @@ describe('Service', () => {
       tmpDirPath = getTmpDirPath();
     });
 
+    it('should call configSchemaHandler.validateConfig', () => {
```

Comment:
Let's not add such tests. They test characteristic and not outcome of internal implementation. IMO it's an anti-pattern which doesn't confirm on bugs while it makes any refactors hard to achieve (when implementation changes tests are forced to be changed too, even though outcome is not changing)